withdraw-by-check/LETTERSPECS/BANNO.CHECK.WITHDRAW.V1.CFG

```diff
@@ -53,7 +53,7 @@ LT: ALL
 *
 * MINIMUM AND MAXIMUM WD AMOUNTS
 *  Line must begin with "MIN:" and/or "MAX:"
-*  Enter the minimum and/or maximum check amount for check withdraws
+*  Enter the minimum and/or maximum check amount for check withdrawls
```

Comment:
Your Code Looks Good, Tom!! Thank you for adding "Minimum WD amount" and "Maximum WD amount" as CU's have been requesting that.

withdrawls should be withdrawals